opencti-platform/opencti-graphql/src/resolvers/indicator.js

```diff
@@ -58,6 +58,10 @@ const indicatorResolvers = {
     basedOn: buildRefRelationKey(RELATION_BASED_ON),
     indicates: buildRefRelationKey(RELATION_INDICATES),
     sightedBy: buildRefRelationKey(STIX_SIGHTING_RELATIONSHIP),
+    creator: 'creator_id',
+  },
+  IndicatorsOrdering: {
+    creator: 'creator_id',
```

Comment:
Do we want to order by creator? Or just filter on it ?

Reply:
Both :)